### PR TITLE
refactor docker and remove unnecessary cookie cutter stuff

### PR DESCRIPTION
**Pull request by MJedr**
* add missing docker-compose services
* simplyfy docker setup
* make possible running tests locally with `poetry run pytest`
* ref: cern-sis/issues-scoap3#137
* ref: cern-sis/issues-scoap3#146

### CODE REVIEW

**Review by PascalEgn — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,83 +3,11 @@
...
-
   # Enable version updates for Python/Pip - Production
   - package-ecosystem: "pip"
     # Look for a `requirements.txt` in the `root` directory
     # also 'setup.cfg', 'runtime.txt' and 'requirements/*.txt'
     directory: "/"
     # Every weekday
     schedule:
-      interval: "daily"
+      interval: "monthly"
```
Comment (drjova) [inline .github/dependabot.yml]: Would that work with `poetry`? I think we should update it

File: .gitignore [hunk 2]
```diff
@@ -280,5 +280,3 @@ scoap3/media/
 scoap3/static/webpack_bundles/
 webpack-stats.json
 .env
-.envs/*
-!.envs/.local/
```
Comment (drjova) [inline .gitignore:284]: why removing it  `.envs` from `.gitignore`?

File: compose/local/django/celery/beat/start [hunk 3]
```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-set -o errexit
-set -o nounset
-
-
-rm -f './celerybeat.pid'
-exec watchfiles celery.__main__.main --args '-A config.celery_app beat -l INFO'
```
Comment (drjova) [inline compose/local/django/celery/beat/start:8]: why don't we keep this files to use them in the docker-compose?

File: .envs/.django [hunk 4]
```diff
@@ -7,8 +7,8 @@ IPYTHONDIR=/app/.ipython
 REDIS_URL=redis://redis:6379/0
 
 # Celery
-# ------------------------------------------------------------------------------
-
+CELERY_BROKER_URL="amqp://guest:guest@localhost:5672/"
+CELERY_RESULT_BACKEND="redis://localhost:6379/2"
```
Comment (drjova) [inline .envs/.django:11]: why we remove them from `.local`?

File: config/wsgi.py [hunk 5]
```diff
@@ -27,7 +27,7 @@
 # if running multiple sites in the same mod_wsgi process. To fix this, use
 # mod_wsgi daemon mode with each site in its own daemon process, or use
 # os.environ["DJANGO_SETTINGS_MODULE"] = "config.settings.production"
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.production")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.local")
```
Comment (drjova) [inline config/wsgi.py]: locally we are using `runserver` there is no need to modify the `wsgi`

File: config/settings/production.py [hunk 6]
```diff
@@ -1,198 +0,0 @@
-import logging
```
Comment (drjova) [inline config/settings/production.py]: we should keep the production settings

File: config/settings/local.py [hunk 7]
```diff
@@ -23,44 +23,6 @@
     }
 }
 
-# EMAIL
-# ------------------------------------------------------------------------------
-# https://docs.djangoproject.com/en/dev/ref/settings/#email-host
-EMAIL_HOST = env("EMAIL_HOST", default="mailhog")
```
Comment (drjova) [inline config/settings/local.py]: no need to remove these, locally we will need the `debug_toolbar` etc

File: .gitignore (hunk 2 above)
Comment (MJedr) [reply .gitignore:284]: cause I moved envfiles to `.envs` from `.envs/.local` and don't see any reason to have it deep in the file structure

File: compose/local/django/celery/beat/start (hunk 3 above)
Comment (MJedr) [reply compose/local/django/celery/beat/start:8]: IMO it's much easier to check what's the entrypoint when adding a command directly in compose

File: config/settings/local.py (hunk 7 above)
Comment (MJedr) [reply config/settings/local.py]: we need an email locally? why?

**Review by drjova — APPROVED** [review]

**Pull request merged**